src/features/customer/checkout/CheckoutView.js

```diff
@@ -0,0 +1,103 @@
+import React, { useEffect } from 'react';
+import { Row, Col, Divider, Typography, Form, message } from 'antd';
+import axios from 'axios';
+import PaymentForm from './components/PaymentForm';
+import AddressForm from './components/AddressForm';
+import Checkout from './components/Checkout'; 
+import useCartStore from 'context/CartStore';
+import ServiceConstants from 'constants/ServiceConstants';
+
+const { Text } = Typography;
+
+const CheckoutView = () => {
+  const { cart, clearCart } = useCartStore(); // Get the cart from the store
+  const selectedProducts = Object.values(cart); // Convert cart object to array
+  const [addressForm] = Form.useForm();
+  const [paymentForm] = Form.useForm();
+
+  useEffect(() => {
```

Comment:
bunu kaldirabiliriz

Reply:
form kısmını kullanıyorum, inputları kontrol etmek için. useEffect'i mi kaldırabiliriz demek istedin yoksa

Reply:
Evet useEffect icin demistim